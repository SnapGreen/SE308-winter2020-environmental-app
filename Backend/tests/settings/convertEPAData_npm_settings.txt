settings check:
	SETTINGS: Backend/scripts/settings_npm.txt
   SCRIPTSDIR: Backend/scripts/
	DATADIR: Backend/data/
	EPADIR: EPA/
	OUTFILE_END: .json
	NEWDATADIR: fakedir/
	EPADATASOURCE: fakefile.xls
	EPA_PATTERNFILE: epa_removalpatterns.txt
	EPA_PATTERN_PATH: Backend/scripts/epa_removalpatterns.txt
	NEWDATAPATH: fakedir/fakefile.xls
	AWKDIR: Backend/scripts/awk/
	TMPDIR: Backend/temp/
	TMPFILE_END: .tmp
	SAFER_PREFIX: fakefile
	SAFERCSVROUGH_TMP: Backend/temp/fakefile_roughcsv.tmp
	SAFERCSVPREPPED_TMP: Backend/temp/fakefile_preppedcsv.tmp
	SAFERSPLIT_PREFIX: Backend/temp/fakefile
	SAFEROLD: Backend/temp/fakefile00
	SAFERNEW: Backend/temp/fakefile01
	SAFEROLD_TMP: Backend/temp/fakefile00.tmp
	SAFERNEW_TMP: Backend/temp/fakefile01.tmp
	SAFEROLDTRIM_TMP: Backend/temp/fakefile_oldtrim.tmp
	SAFERNEWTRIM_TMP: Backend/temp/fakefile_newtrim.tmp
	SAFERJOINED_TMP: Backend/temp/fakefile_joined.tmp
	SAFERB4_TMP: Backend/temp/fakefile_b4.tmp
	SAFERCLEAN_TMP: Backend/temp/fakefile_clean.tmp
	SAFERSORTED_TMP: Backend/temp/fakefile_sorted.tmp
	SAFERUNIQUE_TMP: Backend/temp/fakefile_unique.tmp
	SAFER_JSON: Backend/data/EPA/fakefile.json
	AWK_OLDEPA_TRIM: Backend/scripts/awk/epaoldtrim.awk
	AWK_NEWEPA_TRIM: Backend/scripts/awk/epanewtrim.awk
	AWK_EPA_JSON: Backend/scripts/awk/epatojson.awk
	USAGE:
		Usage: ./convertEPAData.sh [OPTION] (use option -h for help)
	HELP:
		Usage: ./convertEPAData.sh [OPTION] (use option -h for help)
		**If no OPTION supplied, debug mode on (temp files remain)
			-b: bypass debug mode
			-s: output settings only
			-h: print help
