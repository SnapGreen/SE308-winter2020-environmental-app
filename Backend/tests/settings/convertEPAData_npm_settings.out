settings check:
	SETTINGS: Backend/scripts/settings_npm.txt
	SCRIPTSDIR: 
	DATADIR: 
	EPADIR: 
	OUTFILE_END: 
	NEWDATADIR: fakedir/
	EPADATASOURCE: fakefile.xls
	EPA_PATTERNFILE: 
	EPA_PATTERN_PATH: 
	NEWDATAPATH: fakedir/fakefile.xls
	AWKDIR: 
	TMPDIR: 
	TMPFILE_END: 
	SAFER_PREFIX: fakefile
	SAFERCSVROUGH_TMP: fakefile_roughcsv
	SAFERCSVPREPPED_TMP: fakefile_preppedcsv
	SAFERSPLIT_PREFIX: fakefile
	SAFEROLD: fakefile00
	SAFERNEW: fakefile01
	SAFEROLD_TMP: fakefile00
	SAFERNEW_TMP: fakefile01
	SAFEROLDTRIM_TMP: fakefile_oldtrim
	SAFERNEWTRIM_TMP: fakefile_newtrim
	SAFERJOINED_TMP: fakefile_joined
	SAFERB4_TMP: fakefile_b4
	SAFERCLEAN_TMP: fakefile_clean
	SAFERSORTED_TMP: fakefile_sorted
	SAFERUNIQUE_TMP: fakefile_unique
	SAFER_JSON: fakefile
	AWK_OLDEPA_TRIM: epaoldtrim.awk
	AWK_NEWEPA_TRIM: epanewtrim.awk
	AWK_EPA_JSON: epatojson.awk
	USAGE:
		Usage: ./convertEPAData.sh [OPTION] (use option -h for help)
	HELP:
		Usage: ./convertEPAData.sh [OPTION] (use option -h for help)
		**If no OPTION supplied, debug mode on (temp files remain)
			-b: bypass debug mode
			-ns: output settings relative to repo root
			-s: output settings only
			-t: test mode (silent)
			-h: print help
